### PR TITLE
chore(tokens): add tokensToCssVars utility function

### PR DESCRIPTION
**Pull request by mainframev**
[Doc](https://docs.google.com/document/d/13q8UrRqCzvnQ9Pv6wH5QS_7SG4eeC7hdXTtvqNhWc9k/edit?usp=sharing) 
[Slack thread
](https://skypicker.slack.com/archives/C05FDP4UM3N/p1697428801266619)
 Storybook: https://orbit-mainframev-feat-tokens-add-css-variables-utility.surge.sh

### CODE REVIEW

File: packages/orbit-components/src/OrbitProvider/index.tsx [hunk 1]
```diff
@@ -22,6 +23,17 @@ import type { Props } from "./types";
  *
  */
 const OrbitProvider = ({ theme, children, useId }: Props) => {
+  React.useEffect(() => {
+    const ID = "orbit-theme-css-vars";
+    const style = document.createElement("style");
+    style.setAttribute("id", ID);
+    style.innerText = tokensToCssVars({ tokens: theme.orbit }).replace(/\s/g, "");
```
Comment (DSil) [inline packages/orbit-components/src/OrbitProvider/index.tsx:30]: This doesn't work. It needs a selector.

```suggestion
    style.innerText = `
    :root {
      ${tokensToCssVars({ tokens: theme.orbit }).replace(/\s/g, "")}
    }
    `;
```

**Review by DSil — CHANGES_REQUESTED** [review]

File: packages/orbit-components/src/OrbitProvider/index.tsx [hunk 2]
```diff
@@ -22,6 +23,20 @@ import type { Props } from "./types";
  *
  */
 const OrbitProvider = ({ theme, children, useId }: Props) => {
+  React.useEffect(() => {
+    const ID = "orbit-theme-css-vars";
+    const style = document.createElement("style");
+    style.setAttribute("id", ID);
+    style.innerText = tokensToCssVars({ tokens: theme.orbit, cssClass: ":root" }).replace(
+      /\s/g,
+      "",
```
Comment (DSil) [inline packages/orbit-components/src/OrbitProvider/index.tsx:32]: I would prefer this replace to be inside the function, and not here, if possible.

**Review by DSil — COMMENTED** [review]
LGTM, just one question.

Also, this should be a fix commit, no? This is a broken feature we currently have.

**Comment by mainframev** [comment]
> LGTM, just one question.
> 
> Also, this should be a fixed commit, no? This is a broken feature we currently have.

Do you mean the OrbitProvider adjustment? no, it's a chore, for the user API is not changed 🤔 Under the hood are css variables, but they won't need any additional setup.

**Comment by DSil** [comment]
Orbit was broken for WL. This change fixes that problem. Therefore, it's a fix. A fix commit does not require an API change.

**Review by DSil — APPROVED** [review]

**Pull request merged**